x-pack/plugins/cloud_security_posture/public/pages/findings/translations.ts

```diff
@@ -190,3 +186,8 @@ export const PLATFORM = i18n.translate('xpack.csp.findings.platformLabel', {
 export const NO_FINDINGS = i18n.translate('xpack.csp.findings.nonFindingsLabel', {
   defaultMessage: 'There are no Findings',
 });
+
+export const FINDINGS_SEARCH_PLACEHOLDER = i18n.translate(
+  'xpack.csp.findings.searchBar.searchPlaceholder',
+  { defaultMessage: 'Search findings (eg. resource.type : "API Server")' }
```

Comment:
The example is falsely, 
To filter API Server the user would need to apply the following `rule.section: "API Server"`